.bazelrc

```diff
@@ -65,6 +65,8 @@ build:clang-tidy --output_groups=report
 build:gcc --action_env=BAZEL_COMPILER=gcc
 build:gcc --action_env=CC=gcc
 build:gcc --action_env=CXX=g++
+build:gcc --cxxopt -Wno-invalid-offsetof --host_cxxopt -Wno-invalid-offsetof
+build:gcc --cxxopt -Wno-deprecated --host_cxxopt -Wno-deprecated
```

Comment:
As a general rule, those V8 workarounds should be:
1. added to `v8.patch` as patches against V8's `bazel/defs.bzl`,
2. upstreamed to V8, so we don't need to maintain those local workarounds forever.

Especially, because those `cxxopts` won't be propagated to dependent projects (i.e. Envoy).